package.json

```diff
@@ -28,6 +28,7 @@
   ],
   "main": "dist/css-box-model.cjs.js",
   "module": "dist/css-box-model.esm.js",
+  "types": "index.d.ts",
```

Comment:
we publish the `/src` and `/dist` folders (by using `/files`). Can we move the `d.ts` to `/src`?

Reply:
Sure thing 👍

Reply:
updated